distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -927,45 +947,6 @@ type Table
             Table_Data t ->
                 Table_Data (self.java_table.join t drop_unmatched on left_suffix right_suffix)
 
-    ## ALIAS Clean Rows
-
-       Returns a new Table without rows that contained missing values in any of
-       the columns.
-
-       > Example
-         Remove any rows that contain missing values from the table.
-
-             import Standard.Examples
-
-             example_drop_missing_rows =
-                 Examples.inventory_table.drop_missing_rows
-    drop_missing_rows : Table
-    drop_missing_rows self =
-        cols = self.columns
-        case cols.not_empty of
-            True ->
-                any_missing_mask = cols.map .is_missing . reduce (||)
-                non_missing_mask = any_missing_mask.not
-                self.where non_missing_mask
-            False -> self
-
-    ## ALIAS Clean Columns
-
-       Returns a new Table without columns that contained any missing values.
-
-       > Example
-         Remove any columns that contain missing values from the table.
-
-             import Standard.Examples
-
-             example_drop_missing_cols =
-                 Examples.inventory_table.drop_missing_columns
-    drop_missing_columns : Table
-    drop_missing_columns self =
```

Comment:
This function is not accounted for by the new design. Are you sure we want to remove it?

Reply:
Felt a very odd function just to drop columns with NULL values and didn't feel right in it's current form.

It is fairly easy to do:
```python
table.select_column By_Column (table.columns.filter c->(c.contains Nothing).not)
```

Reply:
It was inspired by Pandas' [`dropna`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.dropna.html) (with `axis=1` it does exactly that).

I remember using it sometimes - in some cases, we wanted to care only about columns that contain all values. Of course it's probably better to inspect the column and one may drop them manually - so I'm not saying this function is very needed. But it seems to be used from time to time, so may be worth considering. We can always re-add too it if a need arises.

Reply:
Ok will add it back in and amend design note.